README.md

```diff
@@ -102,7 +101,7 @@ If you would like to use the [Advanced Custom Fields (ACF)](https://www.advanced
 2. In `composer.json` add the following to the `"repositories"` array.
 
    ```json
-   {
+   ({
```

Comment:
I don't know why prettier does this but I had to turn off autoformatting and `--no-verify` my commit to prevent this from happening 🙃